graphistry/pygraphistry.py

```diff
@@ -183,7 +183,7 @@ def sso_login(org_name=None, idp_name=None, sso_timeout=SSO_GET_TOKEN_ELAPSE_SEC
         :param idp_name: Set sso login idp name. Default as None (for site-wide SSO / for the only idp record).
         :type idp_name: Optional[str]
         :param sso_timeout: Set sso login getting token timeout in seconds (blocking mode), set to None if non-blocking mode. Default as SSO_GET_TOKEN_ELAPSE_SECONDS.
-        :type sso_timeout: Optional[int|None]
+        :type sso_timeout: Optional[int]
```

Comment:
@vaimdev  not sure why that didn't go through, maybe python versions -- maybe explicit `Optional[Union[int, None]` ?